projects/packages/identity-crisis/src/class-url-secret.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * IDC URL secret functionality.
+ *
+ * @package  automattic/jetpack-identity-crisis
+ */
+
+namespace Automattic\Jetpack\IdentityCrisis;
+
+use Jetpack_Options;
+use WP_Error;
+
+/**
+ * IDC URL secret functionality.
+ */
+class URL_Secret {
```

Comment:
Minor but why the `URL_` prefix? I think plain `Secret` could also work. Also we could add a DocBlock here explaining it's purpose, eg: "A short lived secret used to verify whether an IDC is coming from the same vs a different Jetpack site"

Reply:
The secret is not multi-purpose (e.g. WP nonce), and used solely for a very specific task. I believe the name should also be more specific than simple `Secret`.
Updated phpdoc: https://github.com/Automattic/jetpack/pull/32005/commits/1f3358423b960f6cf3ebf9b828ccb2d32d607477

Reply:
Agreed, my logic was that we are already under the `Automattic\Jetpack\IdentityCrisis` namespace so the `Secret` is essentially a functionality tight to IDC